Tests/installation_tests/cocoapods/with_frameworks/Podfile

```diff
@@ -1,6 +1,7 @@
 use_frameworks!
 
 target 'CocoapodsTest' do
+	platform :ios, '8.0'
```

Comment:
👍

Reply:
Yeah I noticed our cocoapods test was throwing warnings because we didn't have this specified. Probably need to add it to the without_frameworks version also. I have a task on me to change a bunch of this stuff when we drop Xcode 8 / iOS 8 so I'll do it then.